rpa-home/modules/ROOT/pages/automation-security-securesessions.adoc

```diff
@@ -0,0 +1,50 @@
+= Secure Sessions
+
+In RPA automations, a bot simulates a human user. The bot logs in to a system, sees the screen, and can interact with the UI via key strokes and mouse actions. That way, the bot can even interact with legacy systems, that do not provide an API.
+
+The bot logs in to the computer, where it is installed. But, of course, it does not sit on a chair in front of the desktop. Nevertheless, if you would enter the room where the computer is, you could see what the bot is doing: The screen would show the open applications, the mouse would  move around and typing would appear in forms. In MuleSoft RPA, we call this a _Desktop Session_. To prevent unauthorized data access or manipulation, you would need to guard the physical room.
+
+Or you could use what we call _Secure Sessions_.
+
+Secure Sessions are hidden Desktop Sessions. They run in the background and only the bot can access them. An additional advantage of the Secure Sessions is, that the bot can multiply itsself to run several process instances in parallel.
+
+To use Secure Sessions you must ensure that all prerequisites are met and all responsible persons are informed.
+
+== General Considerations
+
+* At the same time, a bot can only run in Desktop or in Secure Session mode.
```

Comment:
```suggestion
* A bot either can run in Desktop or Secure Session mode.
```

Reply:
Replaced with 
A bot either can run in Desktop Session or Secure Session mode. You can change the mode while no configurations are deployed to the bot.
[While - because it refers to a time span]